.ruby-style.yml

```diff
@@ -1,23 +1,22 @@
 AllCops:
+  RubyInterpreters:
```

Comment:
Commit msg should explain why the previous was wrong.

Looks like it was just _assuming_ all ruby files were included, which I expect was valid at some point in the past, but now rubocop is expecting you you to explicitly include them?